src/display_artwork.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+
+class DisplayArtwork extends Component {
+
+  formatArtistNames(artwork) {
+    if (artwork.artists != null ? artwork.artists[0] : undefined) {
+      let names = artwork.artists.map(artist => artist.name);
+      return names.join(', ');
+    } else {
+      return (artwork.artist != null ? artwork.artist.name : undefined);
+    }
+  }
+
+  renderTitle(artwork) {
+    if (artwork.title) {
+      return <span className='title'><em>{artwork.title}</em></span>
+    }
+  }
+
+  renderDate(artwork) {
+    if (artwork.date) {
+      return <span className='date'>{artwork.date}</span>
+    }
+  }
+
+  renderSpacer(artwork) {
+    if (artwork.title && artwork.date) {
+      return <span className='spacer'>, </span>
+    }
+  }
+
+  renderPartner(artwork) {
+    if (artwork.partner.name) {
+      return <p>{artwork.partner.name}</p>
+    }
+  }
+
+  render() {
+    const { artwork } = this.props;
+    return (
+      <div className='display-artwork'>
+        <img
+          src={artwork.image}
+          className='display-artwork__image' />
+        <div className='display-artwork__caption' style={styles}>
```

Comment:
👍  for inline styles and namespaced CSS here. I'd encourage namespacing the classes above like you did here as well, e.g. `<span className='display-artwork__date'>{artwork.date}</span>`